src/zksync-specifics/examples/smart-account.md

```diff
@@ -129,7 +131,7 @@ contract TwoUserMultisig is IAccount, IERC1271 {
         }
 
         // The fact there is enough balance for the account
-        // should be checked explicitly to prevent user paying for fee for a
+        // should be checked explicitly to prevent the user from paying a fee for a
```

Comment:
Also here @MBerguer I would leave it as it is because this is extracted from this [tutorial](https://code.zksync.io/tutorials/native-aa-multisig)